examples/to_mdbx.rs

```diff
@@ -0,0 +1,150 @@
+//! Convert a block folder to a MDBX folder
+//! For help, run: `cargo run --example to_mdbx -- --help`
+
+use alloy_primitives::{Address, Bytes, FixedBytes, B256, B64, U64};
+use clap::Parser;
+use libmdbx::{
+    Database, DatabaseKind, DatabaseOptions, Mode, NoWriteMap, ReadWriteOptions, SyncMode,
+    TableFlags, WriteFlags,
+};
+use pevm::{EvmAccount, EvmCode};
+use revm::primitives::Bytecode;
+use std::{
+    collections::{BTreeMap, HashMap},
+    fs::File,
+    io::{BufReader, Error},
+    path::Path,
+};
+
+type B416 = FixedBytes<52>;
+type B0 = FixedBytes<0>;
+
+/// Convert a block folder to a MDBX folder
+#[derive(Parser, Debug)]
+#[clap(name = "to_mdbx")]
+struct Args {
+    /// Path to bytecodes.bincode
+    #[clap(long, value_name = "FILE")]
+    bytecodes: String,
+    /// Path to pre_state.json
+    #[clap(long, value_name = "FILE")]
+    pre_state: String,
+    /// Path to block_hashes.json
+    #[clap(long, value_name = "FILE")]
+    block_hashes: Option<String>,
+    /// Path to output MDBX dir
+    #[clap(long, value_name = "DIR")]
+    output: String,
+}
+
+#[derive(Debug, Clone, Default)]
+struct Tables {
+    balance: HashMap<Address, B256>,
```

Comment:
Why don't we just map `Address` to a (serialized) `EvmAccount` on disk? This granularity suggests that we'd need many expensive I/O operations to fetch an account. State commitments will also be very slow. 

Ideally, we would use the least I/O then cache hot accounts in memory anyway.